docs/theme-app-extension.md

```diff
@@ -75,3 +77,17 @@ Then in the html template code portion you'll find the `{%- for block in section
 ```
 
 You should then see your blocks in theme editor under product page product section and if you'll select one of the blocks it will appear in sidepannel where Name, Price and other info about the product is present.
+
+## FAQ
+
+Q: I can't see my app blocks in my Theme Editor.
+
+A: After you use the CLI command `Shopify extension push` you must go to the extension page in the partner dashboard. Create a new version of the extension. Publish the version, to push it live to merchants.
```

Comment:
lower case `s` for the command.

I'll just push a new revision since it's the only issue.